contracts/contracts/gateway/GatewayMessengerFacet.sol

```diff
@@ -13,12 +13,15 @@ import {AssetHelper} from "../lib/AssetHelper.sol";
 import {CrossMsgHelper} from "../lib/CrossMsgHelper.sol";
 import {FvmAddressHelper} from "../lib/FvmAddressHelper.sol";
 
+import {EnumerableSet} from "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+
 string constant ERR_GENERAL_CROSS_MSG_DISABLED = "Support for general-purpose cross-net messages is disabled";
 string constant ERR_MULTILEVEL_CROSS_MSG_DISABLED = "Support for multi-level cross-net messages is disabled";
 
 contract GatewayMessengerFacet is GatewayActorModifiers {
     using FilAddress for address payable;
     using SubnetIDHelper for SubnetID;
+    using EnumerableSet for EnumerableSet.Bytes32Set;
```

Comment:
why not use `EnumerableMap` directly, feels like this is implementing `EnumerableMap`.

Reply:
oh, they dont support the value type.